tests/providers/google/cloud/operators/test_presto_to_gcs_system.py

```diff
@@ -0,0 +1,143 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import os
+
+import pytest
+from psycopg2 import ProgrammingError
```

Comment:
Why to use PostgreSQL adapter package for Presto tests?
You can import `ProgrammingError` from `prestodb`
https://github.com/prestodb/presto-python-client/blob/master/prestodb/exceptions.py#L192

Reply:
I deleted the use of this class in the code.